bundles/org.openhab.binding.mideaac/README.md

```diff
@@ -0,0 +1,121 @@
+## Midea AC Binding
+
+This binding integrates Air Conditioners that use the Midea protocol. Midea is an OEM for many brands.
+
+An AC device is likely supported if it uses one of the following Android apps or it's iOS equivalent.
+
+ ```
```

Comment:
Tables do not need to go in code fences.

Reply:
OK.  However, I had been using the VSC icon to display "how this will look" and without the code fences it all gets squished. I guess that icon doesn't work

Reply:
Done